server.go

```diff
@@ -584,3 +585,19 @@ func runHTTPandHTTPSBandwidthHandler(w http.ResponseWriter, r *http.Request, p E
 		atomic.AddUint64(&test.testResult.data, uint64(r.ContentLength))
```

Comment:
Can we make this conditional so it is incremented only for Bandwidth tests? Otherwise we would get wrong results for latency on server side.

Reply:
Done